### DISCUSSION TITLE
New User - Some Questions

### DISCUSSION THREAD

**Discussion by chart-analyst** [ghostfolio/ghostfolio#2327] [Q&A] [answered]
Hey All, I'm just getting started with Ghostfolio and have some questions.  I started a free cloud account and populated a few accounts to get started and see how things work.  Then, thanks to some kind folks here, I was able to get it running on my Synology NAS. A few questions:

Q1: Can I use my cloud account token to sign in to Ghostfolio on my NAS?  I tried it, token was rejected.  Do I need to create a new account on my NAS?

Q2: What data provider does Ghostfolio use?  Is the data real time? I've tried to find some info on the website, but have yet to find anything. The pricing plans seem to imply real time data is only available for the premium level - is that true?

Q3: Once I get logged in to my self-hosted account, will I need to set up a data provider?

Q4: If I subscribe to premium, will I be able to access premium benefits using my self-hosted system?  Or is premium only for cloud accounts?

I have recently had to move some accounts away from my primary broker.  So now I need a place to consolidate and track my accounts.  This is my primary goal - to see all accounts and track prices, profit & loss, and net worth in one spot in real time.

Thanks!

**Comment by chart-analyst** [comment]
OK, I exported my data from my cloud account, created a new account on my NAS and imported the data into the new NAS account. I also noted that Ghostfolio automatically assigned Yahoo as the data provider - I'm guessing it's 15 minute delayed.  So Q1 and Q3 are answered.

**Comment by FraneCaleta** [comment, marked as answer]
Hi @chart-analyst,

**_Q1: Can I use my cloud account token to sign in to Ghostfolio on my NAS? I tried it, token was rejected. Do I need to create a new account on my NAS?_**
I started with Cloud version first as well. Once I setup my local environment and tried using the cloud token, I'm pretty sure it was rejected/unrecognizable, so I created a new account. Exporting data from cloud to local machine was quite easy process though. I see you've managed to that as well, that's great!

**_Q2: What data provider does Ghostfolio use? Is the data real time? I've tried to find some info on the website, but have yet to find anything. The pricing plans seem to imply real time data is only available for the premium level - is that true?_**
It's using the API from Yahoo as far as I'm aware. It means that data should be real-time updated, possibly with a very small delay. I honestly don't find pricing plan implying that real time data is only available for the premium plan. It's clearly shown that premium and open source accounts have same features (apart from e-mail and chat support). What they mean with "update manually" is that you will have to update your application from time to time (git pull), if you want the latest implemented features, while on premium plan you will not have to worry about that.

**_Q3: Once I get logged in to my self-hosted account, will I need to set up a data provider?_**
No, as I mentioned above, I guess it uses an API to fetch data. No additional data provider set up should be required.

**_Q4: If I subscribe to premium, will I be able to access premium benefits using my self-hosted system? Or is premium only for cloud accounts?_**
Premium is a cloud offering. You won't have to worry about self-hosted anymore, it's always easier to just use cloud offering, since you won't have to worry about manual updates of the application. If I was in your place, I would just use premium to track my portfolio, no need for both, and use self-hosted if I wanted to work on further development of the application.

_If this helped with your questions, please mark it as answered.
If you have any more question, feel free to ask :)_
